### PR TITLE
[ICP-5919] Edited configuration for Ecolink Door/Window Sensor (#3432)

### PR DESCRIPTION
**Pull request by greens**
* Edited configuration for Ecolink Door/Window Sensor

* Fixed initial poll for Ecolink Door/Window Contact.

* Fix for initial closed state.

* Got rid of unnecessary configuration.

* Different refresh for Ecolink device.

* Reduced temperature and increased zoneStatus reports frequency so that periodic zone reports bring device back online instead of periodic temperature reports

* CheckInterval change based on IAS Zone reporting configuration's response instead of temperature's, because in latest commit ZoneStatus report became most frequent.

### CODE REVIEW

**Comment by greens** [comment]
cherry-pick of https://github.com/SmartThingsCommunity/SmartThingsPublic/commit/15b65a6e9c591934291806f41e9eaec85229da2c requested by Shravan

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -183,5 +183,5 @@ def configure() {
 
 	// temperature minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
-	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
+	return refresh() + zigbee.iasZoneConfig(30, 60 * 5) + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 60 * 30) // send refresh cmds as part of config
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:186]: so we want the temperature reporting to change from every 5 min to every 30 min?

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 2]
```diff
@@ -92,12 +92,12 @@ def parse(String description) {
 			} else if (descMap?.clusterInt == 0x0500 && descMap.attrInt == 0x0002) {
 				def zs = new ZoneStatus(zigbee.convertToInt(descMap.value, 16))
 				map = getContactResult(zs.isAlarm1Set() ? "open" : "closed")
-			} else if (descMap?.clusterInt == zigbee.TEMPERATURE_MEASUREMENT_CLUSTER && descMap.commandInt == 0x07) {
+			} else if (descMap?.clusterInt == zigbee.IAS_ZONE_CLUSTER && descMap.commandInt == 0x07) {
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:95]: this replacing the handling for temperature measurement. we don't need that anymore?

Comment (greens) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:95]: this was the handler for the temperature measurement reporting configuration report

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 1 above)
Comment (greens) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:186]: correct

**Pull request merged**